packages/consumption/src/consumption/ConsumptionCoreErrors.ts

```diff
@@ -340,6 +340,13 @@ class Requests {
         return new CoreError("error.consumption.requests.cannotCreateRequestWithExpirationDateInPast", "You cannot create a Request with an expiration date that is in the past.");
     }
 
+    public cannotDeleteIncomingRequestThatIsNotExpired(id: string, status: string) {
```

Comment:
```suggestion
    public canOnlyDeleteIncomingRequestThatIsExpired(id: string, status: string) {
```
Without the negations it's easier to read, but I understand why you used them. So, I'm fine with both.